src/bindings.rs

```diff
@@ -49,7 +49,11 @@ where
             index % 8
         };
         let mask = 1 << bit_index;
-        if val { byte | mask } else { byte & !mask }
```

Comment:
Bindgen also can't seem to decide if it wants this on one line or split across multiple. It flattened this [a few commits ago](https://github.com/libbpf/libbpf-sys/pull/123/files#diff-c030113407a164cbd1ad3500c613f59abd4346ba13ee5b653c56f393c937b0a8R52), only to again explode it in this commit.